sci-electronics/spice/spice-3.5.5-r4.ebuild

```diff
@@ -42,6 +42,7 @@ src_prepare() {
 		conf/linux || die
 	sed -i -e "s:head -1:head -n 1:" util/build || die
 	eapply "${FILESDIR}"/${P}-gcc-4.1.patch
+	eapply "${FILESDIR}"/${P}-arlocal.patch
```

Comment:
Instead of `eapply` we can globally define the array `PATCHES=( )` and then replace `eapply_user` with `default`.

Reply:
Done